_i18n/es.yml

```diff
@@ -145,12 +145,12 @@ security_page:
       los usuarios tengan conocimiento de que su privacidad está siendo protegida
       de la manera que diseñamos la plataforma.'
     p_2: 'Para obtener más información, visite el sitio web de login.gov [Centro de
-      ayuda](site.baseurl /help) o [contáctenos](site.baseurl / contact). También
-      puede visitar nuestro [repositorio de código abierto](https: / /github.com/18F/identity-idp){:target="_blank"}.'
+      ayuda](site.baseurl/help) o [contáctenos](site.baseurl/contact). También
+      puede visitar nuestro [repositorio de código abierto](https://github.com/18F/identity-idp){:target="_blank"}.'
 contact_page:
-  content: 'Primero, vea las respuestas a [preguntas comunes](site.baseurl /help /).
+  content: 'Primero, vea las respuestas a [preguntas comunes](site.baseurl/help/).
     ¿Todavía tiene una pregunta o un problema con la plataforma? Contáctenos a [login-contact@gsa.gov](mailto:
-    login-contact @gsa.gov).'
+    login-contact@gsa.gov).'
```

Comment:
For some reason the line break here is causing a space after the `mailto:`, but bigger question, looks like we are no longer including the email address on the English version, I think this paragraph may be out of sync?  English version:
```First, check out answers to common questions. Still have a question or a problem? Call 844-875-6446 toll-free for more answers and to speak with an information specialist. Information specialists are available between 8:00 a.m. to 8:00 p.m. ET, Monday to Friday, except federal holidays.```